modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -380,6 +410,12 @@ public CouchbaseContainer withFts(boolean fts) {
         return self();
     }
 
+    public CouchbaseContainer withNewFts(String newFts) {
```

Comment:
The way it is handled, it propose a method name like `withFtsFromClasspath`.
Also why `withNewFts` and not `withFts`?